flow/core/experiment.py

```diff
@@ -153,11 +166,19 @@ def rl_actions(*_):
                 vel.append(np.mean(self.env.k.vehicle.get_speed(veh_ids)))
                 ret += reward
 
+                # collect additional information for the data pipeline
+                get_extra_info(self.env.k.vehicle, extra_info, veh_ids, source_id, run_id)
```

Comment:
do we always want the extra info? If so, this is not a data-pipeline specific function, and maybe we should move these methods to flow/utils instead